### ISSUE TITLE
Feature: send several pictures at once

### ISSUE THREAD

**Issue by IpsmLorem** [wireapp/wire-ios#342]
It would be nice to be able to send several pictures at once. It's tedious to send more than a few pictures one by one.

**Comment by david-wire** [comment]
Yes, we have received similar feedback, thanks for the note. No plans to do it immediately, but we will keep that in mind.

**Comment by anatoli26** [comment]
Not having this actually makes sharing media quite uncomfortable. Does it have a high LOE to postpone its implementation?

**Comment by jenskasemets** [comment]
@anatoli26, there are several dependencies for all the features we implement, currently this feature is on our roadmap but not in close future. We will let you know once this is implemented.

**Comment by billypchan** [comment]
Hi @anatoli26, you may share multiple images from Phone app with share extension. We have put this request on our task list. Thanks for the report again!
